src/js/themes/Mender/light.js

```diff
@@ -1,89 +1,135 @@
 import { accordionClasses } from '@mui/material/Accordion';
+import { formLabelClasses } from '@mui/material/FormLabel';
 import { buttonClasses } from '@mui/material/Button';
 import { listItemTextClasses } from '@mui/material/ListItemText';
 
-import { palette, typography, overrides } from './common';
+import { palette as commonPalette, typography, overrides } from './common';
 
-const variantPalette = {
-  ...palette,
+const grey = {
+  '900': '#969696',
+  '800': '#a9a9a9',
+  '700': '#bcbcbc',
+  '600': '#cfcfcf',
+  '500': '#e9e9e9',
+  '400': '#f7f7f7',
+  '300': '#e6f2f1',
+  '200': '#ddedec',
+  '100': '#d8ebe9',
+  '50': '#d4e9e7'
+};
+
+const palette = {
+  ...commonPalette,
   mode: 'light',
-  grey: {
-    '900': '#969696',
-    '800': '#a9a9a9',
-    '700': '#bcbcbc',
-    '600': '#cfcfcf',
-    '500': '#e9e9e9',
-    '400': '#f7f7f7',
-    '300': '#e6f2f1',
-    '200': '#ddedec',
-    '100': '#d8ebe9',
-    '50': '#d4e9e7'
-  },
+  grey,
   background: {
+    light: '#fdfdfd',
+    lightgrey: grey[400],
     default: '#fff',
     dark: 'rgb(50, 50, 50)'
+  },
+  secondary: {
+    lighter: '#8e577b',
+    main: '#5d0f43'
+  },
+  tooltip: {
+    text: grey[50]
+  },
+  text: {
+    primary: 'rgba(10, 10, 11, 0.78)'
```

Comment:
a `text.hint` value of something like `rgba(0, 0, 0, 0.5)` is needed here to ensure the "connector" arrow lines are visible in the update pauses step when creating a deployment. It's also used to give the device list header text a lighter color